### PR TITLE
Configuratooooor

### PR DESCRIPTION
**Pull request by nicovalencia**

### CODE REVIEW

**Review by mckelvey — APPROVED** [review]
💯  @nicovalencia

**Comment by mckelvey** [comment]
Tests appear to be failing on install of `eth-json-rpc-middleware` but vercel can build, so we can circle back on that.

File: components/configurator/ui/ConfiguratorContainer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+
+const ConfiguratorContainer = styled.div`
```
Comment (brolag) [inline components/configurator/ui/ConfiguratorContainer.js:3]: Wondering if we should keep using `rem` iinstead of pixels to keep consistency with the rest of the app.

Comment (nicovalencia) [reply components/configurator/ui/ConfiguratorContainer.js:3]: Ack, yes definitely - I didn't notice we were doing that, sorry!

We can update in a followup branch, or I can tackle this tomorrow AM and the other stories can branch from here for now.

Comment (brolag) [reply components/configurator/ui/ConfiguratorContainer.js:3]: No problem, not a blocker, we can revisit it later. :)

**Pull request merged**